app/Http/Controllers/CommentController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class CommentController extends Controller
+{
+    public function store(Request $request)
+    {
+        $post_id = $this->currentUser()->posts->find($request->post_id)->id;
+        $this->currentUser()->comments()->create([
+            'post_id' => $post_id,
```

Comment:
Viết thế này tốt hơn
```suggestion
        $post = $this->currentUser()->posts->find($request->post_id);
        $this->currentUser()->comments()->create([
            'post_id' => $post->id,
```

Xem thử xem có viết đc kiểu này ko?

```suggestion
        $post = $this->currentUser()->posts->find($request->post_id);
        $this->currentUser()->comments()->create([
            'post' => $post,
```